src/devices/Max7219/MatrixTextWriter.cs

```diff
@@ -0,0 +1,157 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Threading;
+
+namespace Iot.Device.Max7219
+{
+
+    /// <summary>
+    /// Matrix text writer.
+    /// </summary>
+    public class MatrixTextWriter
```

Comment:
Is this the main class that consumers will interact with? or do we expect people to use Max7219 class directly?

Reply:
Depending on the application. If you want to write directly to the matrix, you can use Max7219 Device directly. If you want to write a text into the matrix, you can use the class MatrixTextWriter.